dev/devicelab/bin/tasks/microbenchmarks__preview_dart_2.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
We're running out of horizontal space on the dashboard. Could you please bundle these benchmarks with the non-dart2 versions, and use score keys with a "dart2" suffix?

Reply:
Fixed. PTAL